README.md

```diff
@@ -1 +1,26 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+>   Hello Microverse project aims at testing the micronauts ability to install, configure and use linters.
+    it also tests their ability to document professionally their repositories and proper use if their githubworkflows.
+
+
+## Built With
+
+- HTML
+- CSS
+- Linters
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/rugwizangoga)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/iradukunda-pacific-rugwizangoga)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] It will be better, if you add it licence like [this](https://github.com/microverseinc/readme-template#:~:text=License,is%20MIT%20licensed).